menu.ipxe

```diff
@@ -0,0 +1,17 @@
+#!ipxe
```

Comment:
@miczyg1 definitely yes, but question is how to add it? I will right now just override menu.ipxe, but in future we should think how to fetch changs directy from dasharo/edk2

Reply:
Fixed https://github.com/Dasharo/coreboot/pull/202/commits/a10cb2552ca3402253af11d08065c0e525c95c87

Reply:
It doesn't have to be dasharo/edk2, it may be any other place. The dasahro.ipxe file is added just for reference in edk2 repo just to know what is inside ipxe.efi

Reply:
@miczyg1 what I can do address your concern? Do we have idea how to address things in native way?

Reply:
I would replicate the dasharo.ipxe content in the menu.ipxe file here. I think we will end up with something simialr to what what edk2 dasharo.ipxe has anyways.

Reply:
@miczyg1 isn't it precisely what did in linked commit https://github.com/Dasharo/coreboot/commit/a10cb2552ca3402253af11d08065c0e525c95c87

Reply:
Ahh right. Resolving